alarm_table_defs.hpp

```diff
@@ -71,30 +85,12 @@ class AlarmTableDef
 
   bool is_valid() {return _valid;}
   bool is_not_clear() {return severity() != AlarmDef::CLEARED;}
-
```

Comment:
Did you mean to remove this whitespace? (I think it's quite useful to visually separate the private member variables.)